packages/schematics/angular/migrations/update-8/update-lazy-module-paths_spec.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+// tslint:disable:no-big-function
+import { EmptyTree } from '@angular-devkit/schematics';
+import { SchematicTestRunner, UnitTestTree } from '@angular-devkit/schematics/testing';
+
+describe('Migration to version 8', () => {
+  const schematicRunner = new SchematicTestRunner(
+    'migrations',
+    require.resolve('../migration-collection.json'),
+  );
+
+  let tree: UnitTestTree;
+  const defaultOptions = {};
+  const lazyRoutePath = '/lazy-route.ts';
+  const lazyRoute = `
+
+import { Route } from '@angular/router';
+
+const routes: Array<Route> = [
+  {
+    path: '',
+    loadChildren: './lazy/lazy.module#LazyModule'
+  }
+];
+
+`;
+
+  const lazyChildRoute = `
+
+import { Route } from '@angular/router';
+
+const routes: Array<Route> = [
+  {
+    path: '',
+    children: [{
+      path: 'child',
+      loadChildren: './lazy/lazy.module#LazyModule'
+    }]
+  }
+];
+
+`;
+  const packageJson = {
+    devDependencies: {
+      codelyzer: '^4.5.0',
+    },
+  };
+  const packageJsonPath = '/package.json';
+
+  describe('Migration to import() style lazy routes', () => {
+    beforeEach(() => {
+      tree = new UnitTestTree(new EmptyTree());
+      tree.create(packageJsonPath, JSON.stringify(packageJson, null, 2));
+    });
+
+    it('should replace the module path string', () => {
+      tree.create(lazyRoutePath, lazyRoute);
+      tree = schematicRunner.runSchematic('migration-07', defaultOptions, tree);
```

Comment:
I think your test problem may be here - you should be running `migration-08` since that's where the fixer is registered

Reply:
The other test files in `update-8` also use `migrattion-07` 😞 

```
"migration-07": {
    "version": "8.0.0",
    "factory": "./update-8",
    "description": "Update an Angular CLI project to version 8."
}
```

I've verified that the actual tasks are running and it gets to `context.addTask(new TslintFixTask({`, but then the fix itself isn't run.

I'll keep poking around :)